src/_includes/training_hero.liquid

```diff
@@ -0,0 +1,24 @@
+<section class="training-hero">
+    <div class="training-hero__background-image" style="background-image: url({{ site.baseurl }}{{ page.intro.background-image }})"> </div>
+    <div class="training-hero__lower-background-block"></div>
+    <div class="training-hero__content-container">
+      <div class="training-hero__course-content-info-container">
+        <h1 class="training-hero__title">Crafting Code</h1>
+        <span class="training-hero__duration">Duration: 2 days</span>
+      </div>
+      <div class="training-hero__course-instructor-container">
+        <span class="training-hero__course-instructor-avatar" style="background-image: url({{ site.baseurl }}{{ page.intro.instructor-avatar }})"></span>
```

Comment:
This background doesn't appear.

Reply:
Sigh... not sure why it works locally. Pushing some changes so hopefully that appears